### ISSUE TITLE
New Mitocheck Training Data Confusion

### ISSUE THREAD

**Issue by roshankern** [WayScience/mitocheck_data#21]
Dr. Thomas Walter (from Mitocheck consortium) provided us with the most recent version of the Mitocheck classifier (and the training set used to train this classifier).

This newer dataset has the following structure:

/annotations: XML file for every labeled frame with phenotypic lable of cell, X,Y center coordinates, and Z (likely frame) for each cell.
/controls: folder of images for each labeled frame with labeled cells outline in phenotypic-class specific color
/data:
- features.confusion.txt: confusion matrix of Mitocheck-trained classifier
- features.samples.txt: list of phenotypic class name and coordinate of the annotation (center X,Y) and time (frame).
- features.arff : arff file containing the features of all annotated samples
- features.model : svm model for classification
- features.range : range for feature normalization
/samples: cell images with the same name as features.samples.txt

The confusion with these labels is locating exactly which plate, well, and frame the labeled cells are on. Below is an example line from `features.samples.txt`:

Large	PLLT0010_27--ex2005_05_13--sp2005_03_23--tt17--c5___P00173_01___T00082___X0397___Y0618

The label for this cell (large) and center coordinates (397,618) are very clear. However, the plate, well, and frame are somewhat unclear. This raises the question: how can each cell's plate, well, and frame be determined from a line of the `features.samples.txt` file?

**Comment by roshankern** [comment]
In a future version of `mitocheck_data` we hope to compile the plate, well, frame, and center coordinates for each labeled cell. After changing `IDR_stream` to include an option for returning the outlines of each single cell, we can check if the center coordinates of cells from Mitocheck-labeled training dataset are within the outline of the `IDR_stream` derived cells. If a cell's center is within an outline, we can label it with its Mitocheck label!

**Comment by roshankern** [comment]
Dr. Heriche (from MitoCheck) kindly provided the following information (confirmed by Dr. Walter):

LT0010_27: Plate name/ID (note that the letters LT are part of it)
P00173_01: Position 1 inside well 173 (Mitocheck used only one position per well)
T00082: Time point or frame index. Here I believe this is the frame index. The original Mitocheck data used the time in min from the start (with the first frame at 0 min) so in the original, T is a multiple of 30 (one image every 30 min). Frame indexing shouldn't go past 96 while time points go to 2880.
